server/controllers/ArticleController.js

```diff
@@ -145,6 +146,49 @@ class ArticleController {
         return res.status(200).json();
       }).catch(next);
   }
+
+  /**
+   * this controller handle request for user to get
+   * users feed
+   * @param {Object} req the request body
+   * @param {Object} res the response body
+   * @param {Object} next call the next middle
+   * @returns {Array} an array of articles for users
+   * the user is following order by the latest
+   * article that was created
+   */
+  static getUserFeeds(req, res, next) {
+    let { limit } = req.query;
+    const { id: userId } = req.user;
+    limit = parseInt(limit, 10);
+    Follow
+      .findAll({
+        where: {
+          followerId: userId,
+        },
+        attributes: ['followingId'],
+      })
+      .then((followingArray) => {
+        if (followingArray.length === 0) {
+          return res.status(200).json([]);
```

Comment:
To maintain consistency, I suggest what is returned looks like this `res.status(200).json({ feed: [] })`